libraries/abstractions/secure_sockets/utest/CMakeLists.txt

```diff
@@ -61,7 +63,7 @@
                 "${3rdparty_dir}/lwip/contrib/ports/unix/port/include"
                 "${AFR_ROOT_DIR}/libraries/freertos_plus/standard/tls/include"
                 "${AFR_ROOT_DIR}/libraries/c_sdk/standard/common/include"
-                "${abstraction_dir}/wifi/include"
+                "${CMAKE_CURRENT_LIST_DIR}/include"
```

Comment:
I notice there are two duplicate entries for `"${CMAKE_CURRENT_LIST_DIR}/include"` as well as another for `../include` (probably from experimentation).

Reply:
This path appears multiple times but in different contexts – it is in `real_include_directories` and `test_include_directories` and it is needed in both places (verified after removing/build failure).